kolibri/plugins/device_management/assets/src/constants.js

```diff
@@ -41,6 +41,11 @@ export const TransferTypes = {
   REMOTEIMPORT: 'remoteimport',
 };
 
+export const ApplicationTypes = {
+  KOLIBRI: 'kolibri',
+  STUDIO: 'studio',
```

Comment:
could you add a comment about where are these strings are coming from?

Currently they look like arbitrary comments, but I imagine they actually need to correspond to strings set elsewhere in the app?